### PR TITLE
Formatting Cargo.toml for movementlabsxyz/movement by category with subcategory alphabetical ordering.

### PR DESCRIPTION
**Pull request by BenDavis36**
Cargo.toml required a reordering.

### CODE REVIEW

**Review by l-monninger — APPROVED** [review]
rollup done

File: Cargo.toml [hunk 1]
```diff
@@ -41,211 +34,186 @@ publish = false
...
 revm-primitives = { version = "=2.0.0", default-features = false }
 secp256k1 = { version = "0.27", default-features = false, features = [
     "global-context",
     "rand-std",
     "recovery",
 ] }
-aptos-cached-packages = { git = "https://github.com/movementlabsxyz/aptos-core", rev = "e9b42128f8ed51e90d06beec72d32797693ab66c" }
 
-# celestia dependencies
+## Celestia Sependencies
```
Comment (0xmovses) [inline Cargo.toml:134]:
```suggestion
## Celestia Dependencies
```

File: Cargo.toml [hunk 2]
```diff
@@ -41,211 +34,186 @@ publish = false
...
 celestia-rpc = { git = "https://github.com/eigerco/lumina" }
 celestia-types = { git = "https://github.com/eigerco/lumina" }
 
-# other
+# Other Dependencies
```
Comment (0xmovses) [inline Cargo.toml:138]:
```suggestion

```
Let's leave this as an unnamed category.

**Review by 0xmovses — CHANGES_REQUESTED** [review]

**Review by 0xmovses — APPROVED** [review]
:shipit:

**Pull request merged**